distribution/docs/src/main/resources/_contents/_data/metacard-attributes-table-contents.adoc

```diff
@@ -29,8 +29,8 @@
 |metacard-tags
 |Collections of data that go together, used for filtering
```

Comment:
We probably need a disambiguating note here and in `topic.keyword`. `metacard-tags` is more for internal use, used to distinguish between things like workspaces, history metacards, and the like. Most users are going to want `topic.keyword` instead, or `topic.vocabulary` and `topic.category` tuples for more precise categorization.